src/styles/ProductsCard.css

```diff
@@ -1,48 +1,50 @@
 .card-group {
   width: 80%;
-  justify-content: space-evenly;
+  justify-content: center;
   margin-bottom: 6em;
   margin-top: 3em;
+  display: flex;
+  gap: 89px;
+  flex-wrap: wrap;
 }
 
-
-
-.cardart
-{
-
-  transition: all .5s;
+.cardart {
+  background: #f0f0f0;
+  border-radius: 20px;
+  transition: all 0.5s;
+  max-width: 344px;
```

Comment:
The cards look huge, maybe we can change the width from `344px` to like, ummm maybe `18rem ` so that it looks like normal products card ? 

![image](https://user-images.githubusercontent.com/72851613/206269518-1162ddeb-fade-4117-8b07-cd5b4908d35a.png)

Reply:
i changed it to 18rem